FRED/src/dependencies/scraping/extract.py

```diff
@@ -0,0 +1,28 @@
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import TimeoutException
+class Extract:
+    driver = webdriver.Chrome(ChromeDriverManager().install())
+    def extract(self):
+        driver.get('https://fred.stlouisfed.org/categories/33058')
+        links=['https://fred.stlouisfed.org/series/TERMCBAUTO48NS','https://fred.stlouisfed.org/series/RIFLPBCIANM60NM','https://fred.stlouisfed.org/series/TERMAFCNCNSA']
```

Comment:
The links are hardcoded, this has to be generic that can be used to collect data from any category, spec you need.